doc/sphinx/requirements.txt

```
@@ -0,0 +1,13 @@
+altair>=5.0
+breathe>=4.30
+myst-parser[linkify]>=0.14.0
+sphinx-hoverxref>=0.3b1
+sphinx-design
+sphinx>=5.3,<6
+sphinx_rtd_theme
+sphinxcontrib-bibtex==2.5
+sphinxcontrib-katex
+sphinxcontrib-mermaid
+sphinxcontrib-svg2pdfconverter
+sphinxext-altair
+docutils
```

```
sphinx-rtd-theme 2.0.0 requires docutils<0.21
```
Please add the version compat entry:
```suggestion
docutils<0.21
```